element_lab/lab.py

```diff
@@ -19,44 +23,68 @@ def activate(schema_name: str, create_schema: bool = True, create_tables: bool =
     )
 
 
+@schema
+class Organization(dj.Manual):
+    """Top-level list of all organizations involved in any of the projects.
+
+    Attributes:
+        organization ( varchar(24) ): Abbreviated organization name.
+        organization_name ( varchar(255) ): Full organization name.
```

Comment:
Previously, @dimitri-yatsenko mentioned shortening attribute names like this (e.g., `organization_name` -> `org_name`). We may want to do this now if so.

Reply:
Sure, I can make that adjustment